setup.py

```diff
@@ -84,6 +84,8 @@ def __str__(self):
 libraries = ["archive", "solv", "solvext", "reproc++", CURL_LIB, CRYPTO_LIB]
 if sys.platform == "win32":
     libraries.append("advapi32")
+else:
```

Comment:
actually we should use this new code only in micromamba. for compatibility it's much better to use the conda code for now.

Reply:
alright! I'll have a look at this!

Reply:
I renamed the `yaml-cpp` preprocessor guard introduced for the same purpose `UMAMBA_ONLY` and used it here to keep virtual packages in micromamba and tests only.
Let me know if it looks ok @wolfv

Reply:
I think this needs to go